loopchain/blockchain/blocks/v0_3/block_verifier.py

```diff
@@ -113,14 +115,22 @@ def verify_invoke(self, builder: 'BlockBuilder', block: 'Block', prev_block: 'Bl
                                      f"Expected({new_block.header.state_hash}).")
             self._handle_exception(exception)
         if header.next_reps_hash != new_block.header.next_reps_hash:
-            exception = RuntimeError(f"Block({header.height}, {header.hash.hex()}, "
-                                     f"NextRepsHash({header.next_reps_hash}), "
-                                     f"Expected({new_block.header.next_reps_hash}).")
-            self._handle_exception(exception)
+            if not new_block.header.prep_changed \
+                    and header.next_reps_hash == new_block.header.revealed_next_reps_hash:
+                pass
+            else:
```

Comment:
I think `if (condition): else` statements can be replaced by the following condition.
```suggestion
            if (new_block.header.prep_changed or header.next_reps_hash != new_block.header.revealed_next_reps_hash):
```

Reply:
Good suggestion.